code/scripts/randomFunctions.js

```diff
@@ -0,0 +1,19 @@
+// RNG functions (used throughout the project)
+export const d20 = () => {
+  return Math.floor(Math.random() * 20 + 1);
+};
+export const d10 = () => {
+  return Math.floor(Math.random() * 10 + 1);
+};
+export const d8 = () => {
+  return Math.floor(Math.random() * 8 + 1);
+};
+export const d6 = () => {
+  return Math.floor(Math.random() * 6 + 1);
+};
+export const d4 = () => {
+  return Math.floor(Math.random() * 4 + 1);
+};
+export const random = (maxNumber) => {
+  return Math.floor(Math.random() * maxNumber);
+};
```

Comment:
Why did you repeat this logic so many times? The only one you need is this last one when you repair it by adding 1 or change floor to ceil 🧙🏻